extensions/mailer/runtime/src/main/java/io/quarkus/mailer/ReactiveMailer.java

```diff
@@ -4,8 +4,11 @@
 
 /**
  * A mailer to send email asynchronously.
+ * 
+ * @deprecated Use {@link io.quarkus.mailer.mutiny.ReactiveMailer}.
  */
-public interface ReactiveMailer {
+@Deprecated
+public interface ReactiveMailer extends io.quarkus.mailer.axle.ReactiveMailer {
 
     /**
      * Sends the passed emails.
```

Comment:
The `CompletionStage<Void> send(Mail... mails);` method from this interface could be deleted as it is already defined in the super interface.